ring/util_test.go

```diff
@@ -240,6 +261,98 @@ func TestWaitRingStabilityShouldReturnErrorIfMaxWaitingIsReached(t *testing.T) {
 	assert.InDelta(t, maxWaiting, elapsedTime, float64(2*time.Second))
```

Comment:
[nit] I think we can assert on elapsedTime >= maxWaiting instead of using delta, because we expect at least maxWaiting time has been waited for.

_Same comment applies to tests below._

Reply:
yes